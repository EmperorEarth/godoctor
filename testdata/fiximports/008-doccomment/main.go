package main

// This is the main function.
// The import block should be added above this.
func main() {
  fmt.Println("Hello, Go") // <<<<< fiximports,1,1,1,1,pass
}
