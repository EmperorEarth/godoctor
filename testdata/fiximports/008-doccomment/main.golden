package main

// This is the main function.
// The import block should be added above this.
import (
	"fmt"
)

func main() {
  fmt.Println("Hello, Go") // <<<<< fiximports,1,1,1,1,pass
}
